packages/flutter/test/widgets/root_restoration_scope_test.dart

```diff
@@ -362,7 +362,7 @@ void main() {
     expect(state.bucket, isNotNull);
 
     binding.restorationManager.rootBucket = SynchronousFuture<RestorationBucket>(null);
-    root..decommission()..dispose();
+    root.dispose();
```

Comment:
Why did the one above move to after the pump, and this one didn't?

Reply:
Moved both to after.